src/main/java/io/cdap/plugin/http/common/BaseHttpConfig.java

```diff
@@ -68,6 +68,13 @@ public abstract class BaseHttpConfig extends ReferencePluginConfig {
     public static final String PROPERTY_AUTO_DETECT_VALUE = "auto-detect";
 
     public static final String PROPERTY_SERVICE_ACCOUNT_SCOPE = "serviceAccountScope";
+    public static final String PROPERTY_WAIT_TIME_BETWEEN_PAGES = "waitTimeBetweenPages";
+
+    @Name(PROPERTY_WAIT_TIME_BETWEEN_PAGES)
+    @Nullable
+    @Description("Time in milliseconds to wait between HTTP requests.")
```

Comment:
Add `Defaults to 0` as per the plugin `readme`.

Reply:
Added !